app/src/main/java/io/homeassistant/companion/android/widgets/button/ButtonWidget.kt

```diff
@@ -228,6 +233,13 @@ class ButtonWidget : AppWidgetProvider() {
             // Render the icon into the Button's ImageView
             setImageViewBitmap(R.id.widgetImageButton, icon.toBitmap(width, height))
 
+            widget?.iconColor?.let {
+                if (it.isNotBlank()) {
+                    setInt(R.id.widgetImageButton, "setColorFilter", Color.parseColor(it))
+                    setInt(R.id.widgetImageButtonBackground, "setColorFilter", Color.parseColor(it))
```

Comment:
This will throw an exception if the input is not valid and crash:
```
2024-08-27 12:12:31.708   747-747   AndroidRuntime          pid-747                              E  FATAL EXCEPTION: main
                                                                                                    Process: io.homeassistant.companion.android.debug, PID: 747
                                                                                                    java.lang.IllegalArgumentException: Unknown color
                                                                                                    	at android.graphics.Color.parseColor(Color.java:1399)
                                                                                                    	at io.homeassistant.companion.android.widgets.button.ButtonWidget.getWidgetRemoteViews(ButtonWidget.kt:238)
                                                                                                    	at io.homeassistant.companion.android.widgets.button.ButtonWidget.access$getWidgetRemoteViews(ButtonWidget.kt:47)
                                                                                                    	at io.homeassistant.companion.android.widgets.button.ButtonWidget$updateAllWidgets$1.invokeSuspend(ButtonWidget.kt:114)
                                                                                                    	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
                                                                                                    	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:959)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:100)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:232)
                                                                                                    	at android.os.Looper.loop(Looper.java:317)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8592)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
                                                                                                    	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@61859a4, Dispatchers.Main]
```